tests/unit_tests/model/test_position_pyo3.py

```diff
@@ -91,6 +91,7 @@ def test_position_to_from_dict():
                 "commission": "2.00 USD",
                 "currency": "USD",
                 "event_id": "038990c6-19d2-b5c8-37a6-fe91f9b7b9ed",
+                "info": {},
```

Comment:
Are we adding an info field here for `Position`? I don't see this on the Cython side

Reply:
Yes, we are. Check the `to_dict` method of `OrderFilled` in cython, https://github.com/nautechsystems/nautilus_trader/blob/2526448a5d9b2783b78d6468de31a81b7af992ef/nautilus_trader/model/events/order.pyx#L4800

Reply:
Thats why its here to match Cython basically

Reply:
Isn't that `info` for the individual order fill though, what will go into `info` for the entire `Position`?

Reply:
Yes, this in `info` on order event not on the `Position`